docs/op-stack/src/docs/build/getting-started.md

```diff
@@ -420,46 +451,63 @@ Once you have multiple nodes, it makes sense to use these command line parameter
 
-1. Head over to the `op-batcher` package inside the Optimism Monorepo:
+It is best to give the `Batcher` at least 1 Goerli ETH to ensure that it can continue operating without running out of ETH for gas.
 
-    ```bash
-    cd ~/optimism/op-batcher
-    ```
 
-1. And run the `op-batcher` using the following command. 
-   Replace `<RPC>` with your L1 node URL and replace `<BATCHERKEY>` with the private key for the `Batcher` account that you created and funded earlier. 
-   It’s best to give the `Batcher` at least 1 Goerli ETH to ensure that it can continue operating without running out of ETH for gas.
+```bash
+cd ~/optimism/op-batcher
+
+./bin/op-batcher \
+    --l2-eth-rpc=http://localhost:8545 \
+    --rollup-rpc=http://localhost:8547 \
+    --poll-interval=1s \
+    --sub-safety-margin=6 \
+    --num-confirmations=1 \
+    --safe-abort-nonce-too-low-count=3 \
+    --resubmission-timeout=30s \
+    --rpc.addr=0.0.0.0 \
+    --rpc.port=8548 \
+    --rpc.enable-admin \
+    --max-channel-duration=1 \
+    --l1-eth-rpc=$L1_RPC \
+    --private-key=$BATCHER_KEY
+```
 
-    ```bash
-    ./bin/op-batcher \
-        --l2-eth-rpc=http://localhost:8545 \
-        --rollup-rpc=http://localhost:8547 \
-        --poll-interval=1s \
-        --sub-safety-margin=6 \
-        --num-confirmations=1 \
-        --safe-abort-nonce-too-low-count=3 \
-        --resubmission-timeout=30s \
-        --rpc.addr=0.0.0.0 \
-        --rpc.port=8548 \
-        --rpc.enable-admin \
-        --max-channel-duration=1 \
-        --target-l1-tx-size-bytes=2048 \
-        --l1-eth-rpc=<RPC> \
-        --private-key=<BATCHERKEY>
-    ```
+::: tip Controlling batcher costs
+
+The `--max-channel-duration=n` setting tells the batcher to write all the data to L1 every `n` L1 blocks. 
+When it is low, transactions are written to L1 frequently, withdrawals are quick, and other nodes can synchronize from L1 fast.
+When it is high, transactions are written to L1 less frequently, and the batcher spends less ETH.
 
-   ::: tip Controlling batcher costs
+:::
+
+### `op-proposer`
 
-   The `--max-channel-duration=n` setting tells the batcher to write all the data to L1 every `n` L1 blocks. 
-   When it is low, transactions are written to L1 frequently, withdrawals are quick, and other nodes can synchronize from L1 fast.
-   When it is high, transactions are written to L1 less frequently, and the batcher spends less ETH.
+Now start `op-proposer`, which proposes new state roots.
 
-   :::
+```bash
+cd ~/optimism/op-proposer
+
+./bin/op-proposer \
+    --poll-interval 12s \
+    --rpc.port 8560 \
+    --rollup-rpc http://localhost:8547 \
+    --l2oo-address $L2OO_ADDR \
+    --private-key $PROPOSER_KEY \
+    --allow-non-finalized \
```

Comment:
Including this by default might be a bit of a footgun

Reply:
Even with the comment below

Reply:
You mean `--allow-non-finalized`? I worry that if I don't we'll see people unhappy that `op-proposer` is so slow.

Reply:
OK, I'll remove it.